api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -13,8 +13,8 @@ A user can click the **Export** button to save an Excel file with the exported d
 
 The following instructions show how to enable and configure client-side export:
 
-1. **Reference the required libraries**  
-    Reference or import the <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v3.3.1 or newer and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> libraries. Export also requires the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip</a> library. In a modular environment, this library is listed in package dependencies and is already added. If your app does not use modules, reference JSZip manually.
+1. **Install or reference the required libraries**  
+    This feature requires <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v3.3.1+ and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> v2.0.2+ for all frameworks except Angular. The latter requires `ExcelJS` v1.7.0.
```

Comment:
How about we make it a list like here?
https://github.com/DevExpress/devextreme-documentation/pull/1251/files#diff-d384da97e40226a28ff7c62338331f51R5